public/app/features/dashboard-scene/scene/Scopes/ScopesDashboardsScene.tsx

```diff
@@ -0,0 +1,138 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { from, Subscription } from 'rxjs';
+
+import { GrafanaTheme2, Scope, ScopeDashboardBinding, urlUtil } from '@grafana/data';
+import { SceneComponentProps, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+import { CustomScrollbar, Icon, Input, LoadingPlaceholder, useStyles2 } from '@grafana/ui';
+import { useQueryParams } from 'app/core/hooks/useQueryParams';
+import { t } from 'app/core/internationalization';
+
+import { fetchDashboards } from './api/dashboards';
+
+export interface ScopesDashboardsSceneState extends SceneObjectState {
+  dashboards: ScopeDashboardBinding[];
+  filteredDashboards: ScopeDashboardBinding[];
+  isLoading: boolean;
+  searchQuery: string;
+}
+
+export class ScopesDashboardsScene extends SceneObjectBase<ScopesDashboardsSceneState> {
+  static Component = ScopesDashboardsSceneRenderer;
+
+  private fetchSub: Subscription | undefined;
+
+  constructor() {
+    super({
+      dashboards: [],
+      filteredDashboards: [],
+      isLoading: false,
+      searchQuery: '',
+    });
+
+    this.addActivationHandler(() => {
+      return () => {
+        this.fetchSub?.unsubscribe();
+      };
+    });
+  }
+
+  public fetchDashboards(scopes: Scope[]) {
+    this.fetchSub?.unsubscribe();
+
+    if (scopes.length === 0) {
+      return this.setState({ dashboards: [], filteredDashboards: [], isLoading: false });
+    }
+
+    this.setState({ isLoading: true });
+
+    this.fetchSub = from(fetchDashboards(scopes)).subscribe((dashboards) => {
```

Comment:
I don't think you need this subscription, why not just await `fetchDashboards` call?

Reply:
I guess the benefit is that you can cancel the query automatically when closing the drawer?